binding_generator.py

```diff
@@ -899,6 +908,11 @@ def generate_builtin_class_source(builtin_api, size, used_classes, fully_used_cl
 
     copy_constructor_index = -1
 
+    result.append(f"{class_name}::{class_name}(const Variant *p_variant) {{")
```

Comment:
I see two opening braces, but only one closing.  Is the other closing bracket expected from the next section?

Reply:
The `{{` is how you escape a left curly bracket in a Python f-string. So, it's just a single `{` really.

Reply:
My mistake. 👍